src/pipeline/src/manager/table.rs

```diff
@@ -230,7 +216,7 @@ impl PipelineTable {
                 rows: vec![Row {
                     values: vec![
                         ValueData::StringValue(name.to_string()).into(),
-                        ValueData::StringValue(schema.to_string()).into(),
+                        ValueData::StringValue(DEFAULT_SCHEMA_NAME.to_string()).into(),
```

Comment:
Do you think if we can just use empty field here? This will prevent confusion in long term.

Reply:
The format of the cache key is `{schema}/{pipeline_name}/{version}`, using an empty string seems strange in here.